tasks/llama2-lora/main.py

```diff
@@ -0,0 +1,98 @@
+# Template is based on https://deci.ai/blog/fine-tune-llama-2-with-lora-for-question-answering/
+
+from typing import Any
+
+import logging
+
+from datasets import load_dataset
+from trl import SFTTrainer
+from transformers import pipeline
+from coretex import currentTaskRun, CustomDataset
+
+from src.model import getModelName, loadModel, loadTokenizer, uploadModel
+from src.configurations import getQuantizationConfig, getPeftParameters, getTrainingParameters
+
+
+def loadDataFromCoretex(dataset: CustomDataset) -> Any:
+    dataset.download()
+    for sample in dataset.samples:
+        sample.unzip()
+
+        ######                                        ######
+        # Load data and perform preprocessing as necessary #
+        ######                                        ######
+
+        # Example for loading parquet files from a single sample
+        dataFiles = list(sample.path.glob("*.parquet"))
+        trainingData = load_dataset("parquet", data_files = [str(filePath)for filePath in dataFiles])["train"]
+
+        return trainingData
```

Comment:
This is loading only the first sample in for loop, why? Also return type of the function is `Any`, can it be changed to a real type?

Reply:
It's part of the "Example for loading parquet files from a single sample".
Return will be changed to Dataset form datasets.